Source/WebCore/page/TextIndicator.h

```diff
@@ -144,6 +145,7 @@ class TextIndicator : public RefCounted<TextIndicator> {
     WEBCORE_EXPORT static Ref<TextIndicator> create(const TextIndicatorData&);
     WEBCORE_EXPORT static RefPtr<TextIndicator> createWithSelectionInFrame(Frame&, OptionSet<TextIndicatorOption>, TextIndicatorPresentationTransition, FloatSize margin = FloatSize(defaultHorizontalMargin, defaultVerticalMargin));
     WEBCORE_EXPORT static RefPtr<TextIndicator> createWithRange(const SimpleRange&, OptionSet<TextIndicatorOption>, TextIndicatorPresentationTransition, FloatSize margin = FloatSize(defaultHorizontalMargin, defaultVerticalMargin));
+    WEBCORE_EXPORT static RefPtr<TextIndicator> createWithNode(Node&, TextIndicatorPresentationTransition, FloatSize margin = FloatSize(defaultHorizontalMargin, defaultVerticalMargin));
```

Comment:
Hm...is there a reason why we're generating text indicator options in `createWithNode()`, rather than taking it as an argument? (I think in general, it makes sense to let `TextIndicator` clients to specify options when creating the text indicator).